### ISSUE TITLE
Remove regexp stuff in rename-method

### ISSUE THREAD

**Issue by iangreenleaf** [iangreenleaf/Scisr#25]
Patch attached. This is on hold until we resolve #24.

**Comment by iangreenleaf** [comment]
```
diff --git a/Scisr.php b/Scisr.php
index c83f948..6307398 100644
--- a/Scisr.php
+++ b/Scisr.php
@@ -153,15 +153,12 @@ class Scisr
         $this->_listeners[] = new Scisr_Operations_RenameMethod($class, $oldMethod, $newMethod);

         // Look for matches in comments and strings
+        $this->_listeners[] = new Scisr_Operations_ChangeCommentWords($oldMethod, $newMethod);
+        $this->_listeners[] = new Scisr_Operations_ChangeStringWords($oldMethod, $newMethod);
         $fullOldString = "$class(->|::)$oldMethod";
         $fullNewString = "$class\\1$newMethod";
         $this->_listeners[] = new Scisr_Operations_ChangeCommentWords($fullOldString, $fullNewString, false);
         $this->_listeners[] = new Scisr_Operations_ChangeStringWords($fullOldString, $fullNewString, false);
-        // We have to make sure to avoid double-matching
-        $oldString = "([^>:\w])$oldMethod";
-        $newString = "\\1$newMethod";
-        $this->_listeners[] = new Scisr_Operations_ChangeCommentWords($oldString, $newString);
-        $this->_listeners[] = new Scisr_Operations_ChangeStringWords($oldString, $newString);

         $this->_firstPassRequired = true;
     }
```

**Comment by iangreenleaf** [comment]
Get rid of complicated regexp (closed by 63ea8c29fe44bbc1f5837aadb4270c83c246a0fb)
